Jenkinsfile

```diff
@@ -270,778 +314,414 @@ pipeline {
+                        buildHipClangJobAndReboot(compiler: 'g++', setup_flags: Full_test_limit, build_install: "true")
                     }
                 }
                 stage('Bf16 Hip Install All gfx908') {
                     agent{ label rocmnode("gfx908") }
-                    environment{
-                        cmd = """
-                            ulimit -c unlimited
-                            cd build
-                            CXX=/opt/rocm/llvm/bin/clang++ cmake -DMIOPEN_TEST_BFLOAT16=On -DMIOPEN_TEST_GFX908=On -DMIOPEN_TEST_ALL=On -DBUILD_DEV=Off -DCMAKE_INSTALL_PREFIX=../install -DCMAKE_BUILD_TYPE=release -DMIOPEN_TEST_FLAGS='--verbose --disable-verification-cache' ..
-                            MIOPEN_LOG_LEVEL=5 CTEST_PARALLEL_LEVEL=4 MIOPEN_CONV_PRECISE_ROCBLAS_TIMING=0 make -j\$(nproc) install check
-                        """
-                    }
                     steps{
-                        script{
-                            runDockerJob(cmd: cmd, gpu_arch: "gfx908")
-                        }
+                        buildHipClangJobAndReboot(setup_flags: Bf16_flags + Full_test_limit + gfx908_test, build_install: "true", gpu_arch: "gfx908")
                     }
                 }
             }
         }
-        stage("Full tests II"){
-            when { expression { params.FULL_TESTS } }
+
+        stage("Full Tests II"){
+            when { expression { params.FULL_TESTS && !params.DISABLE_ALL_STAGES } }
+            environment{
+                WORKAROUND_iGemm_936 = " MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_HIP_FWD_V4R1=0"
+            }
             parallel{
-                stage('Fp32 OpenCL Install All') {
-                    agent{ label rocmnode("vega") }
-                    steps{
-                        script{
-                            runDockerJob(compiler: 'g++', flags: '-DBUILD_DEV=Off -DMIOPEN_TEST_ALL=On -DCMAKE_BUILD_TYPE=release')
-                        }
-                    }
-                }
                 stage('Fp32 Hip All gfx908') {
                     agent{ label rocmnode("gfx908") }
-                    environment{
-                        cmd = """
-                            ulimit -c unlimited
-                            cd build
-                            CXX=/opt/rocm/llvm/bin/clang++ cmake -DMIOPEN_TEST_GFX908=On -DMIOPEN_TEST_ALL=On -DBUILD_DEV=On -DCMAKE_BUILD_TYPE=release -DMIOPEN_TEST_FLAGS='--verbose --disable-verification-cache' ..
-                            MIOPEN_LOG_LEVEL=5 CTEST_PARALLEL_LEVEL=4 MIOPEN_CONV_PRECISE_ROCBLAS_TIMING=0 make -j\$(nproc) check
-                        """
-                    }
                     steps{
-                        script{
-                            runDockerJob(cmd: cmd, gpu_arch: "gfx908")
-                        }
+                        buildHipClangJobAndReboot(setup_flags: Full_test_limit + gfx908_test, build_env: WORKAROUND_iGemm_936, gpu_arch: "gfx908")
```

Comment:
@shurale-nkn WORKAROUND_iGemm_936 is for HALF and not needed for FP32 stages. I will fix this soon.